docs/configuration-files/edit-configuration-file.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Edit a Configuration File
+intro: Editing configuration files is crucial for ensuring accurate and up-to-date information is included in the resource deployment process, as needed.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+2. Find the `Applications` card
+3. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application with the configuration file you want to edit and click on`DETAILS`
```

Comment:
Likewise, `on` preposition here is not adhering to our standards.

Reply:
I agree, just removed the 'on'.